bin/millau/runtime/src/rialto_messages.rs

```diff
@@ -177,11 +175,7 @@ impl messages::ThisChainWithMessages for Millau {
 pub struct Rialto;
 
 impl messages::ChainWithMessages for Rialto {
-	type Hash = bp_rialto::Hash;
-	type AccountId = bp_rialto::AccountId;
-	type Signer = bp_rialto::AccountSigner;
-	type Signature = bp_rialto::Signature;
-	type Balance = bp_rialto::Balance;
+	type Chain = bp_rialto::Rialto;
```

Comment:
@svyatonik 
now, we could also remove all specific implementations for `messages::BridgedChainWithMessages::fn maximal_extrinsic_size()`

and we can have just one default generic implementation in `messages::BridgedChainWithMessages` which can use `Chain::max_extrinsic_size()` e.g.:

```
pub trait BridgedChainWithMessages: ChainWithMessages {

	fn maximal_extrinsic_size() -> u32 {
		Self::Chain::max_extrinsic_size()
	}
```

Reply:
@svyatonik 
I see `BridgedChainWithMessages::maximal_extrinsic_size` is used in one place:
```
    pub fn maximal_message_size<B: MessageBridge>() -> u32 {
		super::target::maximal_incoming_message_size(BridgedChain::<B>::maximal_extrinsic_size())
	}
```

is there any scenario, when somebody wants to override in `BridgedChainWithMessages::maximal_extrinsic_size()` what is configured on `Chain::max_extrinsic_size()` ??
if so, maybe we dont even need `BridgedChainWithMessages::maximal_extrinsic_size()`

Reply:
No, I don't think so - `Chain::max_extrinsic_size` should work here. I'll add a fix